### ISSUE TITLE
Create option to use link style pagination rather than the numbered pages

### ISSUE THREAD

**Issue by jackbaty** [xianmin/hugo-theme-jane#2]
Pagination links were recently changed so that they show numbered pages rather than the original "Previous" and "Next" links. This doesn't work as well when there are a large number of pages. I've attached a screenshot of what it looks like on my blog's home page. That's a lot of page links :). I preferred the old navigation.

Perhaps an option for choosing the original link-only or numbered pagination links could be added.

![pagination](https://user-images.githubusercontent.com/970/37245243-3543f624-2463-11e8-92bd-404af4b298a9.png)

**Comment by xianmin** [comment]
@jackbaty  It's amazing! I did not expect that someone could have so many pages.

I fix this problem in two ways:
1. I limit the numbers of pages in the default numbered pagination. 
![deepinscreenshot_select-area_20180311131538](https://user-images.githubusercontent.com/2177570/37250513-e7ca9862-2539-11e8-99dd-8ef40910069d.png)

2. I add an option `paginateOriginalStyle` , if true, use original "Previous" and "Next" links.
In other words, If you preferred the old navigation,  add `paginateOriginalStyle = true` below the `[params]`

**Comment by xianmin** [comment]
I visit your blog.
Would you mind if I put your blog in the README.md theme preview section? :smile:

**Comment by jackbaty** [comment]
This is brilliant, thank you! 

I'm really enjoying your theme. Of course you're welcome to add it to the readme if you like.

**Comment by xianmin** [comment]
@jackbaty  I am so honored that you like this theme.
